model/registry/db.yaml

```diff
@@ -310,6 +310,10 @@ groups:
               value: 'trino'
               brief: 'Trino'
               stability: experimental
+            - id: vector
```

Comment:
Isn't just `vector` a bit off in this list?

I'd expect the `db.system` to either one of the well known dbs - like `db.system: elasticsearch` could be used when elasticsearch is used as a vector db. So mostly this will be the specific db system, right?

And then there are other cases, when the db system may not be (yet) on this list. In that case, similar to `other_sql` wouldn't `other_vector` be a better fit?

Reply:
I agree, `other_vector` is a better fit here.

Reply:
see my comment on the `db.system` above